packages/Teams/Microsoft.Bot.Components.Teams.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version Condition=" '$(PreviewPackageVersion)' == '' ">$(LocalPackageVersion)</Version>
+    <Version Condition=" '$(PreviewPackageVersion)' != '' ">$(PreviewPackageVersion)</Version>
+    <PackageVersion Condition=" '$(PreviewPackageVersion)' == '' ">$(LocalPackageVersion)</PackageVersion>
+    <PackageVersion Condition=" '$(PreviewPackageVersion)' != '' ">$(PreviewPackageVersion)</PackageVersion>
+    <Configurations>Debug;Release</Configurations>
+    <DocumentationFile>bin\$(Configuration)\netstandard2.0\Microsoft.Bot.Builder.Dialogs.Adaptive.xml</DocumentationFile>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageId>Microsoft.Bot.Components.Teams</PackageId>
+    <Description>This library implements .NET support for adaptive dialogs with Microsoft Teams</Description>
+    <Summary>This library implements .NET support for adaptive dialogs with Microsoft Teams</Summary>
+    <ContentTargetFolders>content</ContentTargetFolders>
+    <PackageTags>msbot-component;msbot-action;msbot-trigger</PackageTags>
```

Comment:
msbot-component tag is included in directory.build.props

Reply:
thanks for reviewing

removed 'msbot-component'